.github/workflows/ci.yml

```diff
@@ -257,7 +272,7 @@ jobs:
   #          cri-containerd.test.exe -cri-endpoint="npipe://./pipe/containerd-containerd" -feature="WCOWProcess" -feature="HostProcess"
 
   build:
-    needs: [test, integration-tests]
+    needs: [test-windows, test-linux, integration-tests]
```

Comment:
I feel like `test-linux` should be a require for the `build_gcs` step

Reply:
as it is right now they're not really related... do you want me to move out the testing phase from `build_gcs` here instead, this way the dependency will make more sense.

Reply:
Updated requirements for `build_gcs` job